nbs/core.ipynb

```diff
@@ -35,7 +35,7 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    df1_train = AirPassengersPanel_train[AirPassengersPanel_train['unique_id'] == 'Airline1']
</pre><p>Please use <code>AirPassengersPanel_train.to_parquet('data', partition_cols=['unique_id'], index=False)</code> instead of manually creating the directory to ensure compatibility. You can then define <code>data_directory = list(Path('data').iterdir())</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>